src/meta/src/rpc/service/notification_service.rs

```diff
@@ -84,6 +84,16 @@ where
         let creating_tables = catalog_guard.database.list_creating_tables();
         let users = catalog_guard.user.list_users();
 
+        let context_id = match self
```

Comment:
Could add worker_id in `message SubscribeRequest` to avoid this get.

Reply:
> Could add worker_id in `message SubscribeRequest` to avoid this get.

fixed